client/src/components/EditReviewForm.js

```diff
@@ -0,0 +1,68 @@
+import React from "react"
+import { useState } from "react"
+
+const EditReviewForm = (props) => {
+  const [editedReview, setEditedReview] = useState({
+    title: props.title,
+    body: props.body,
+    rating: props.rating,
+  })
+
+  const handleChange = (event) => {
+    setEditedReview({
+      ...editedReview,
+      [event.currentTarget.name]: event.currentTarget.value,
+    })
+  }
+
+  const handleSubmit = async (event) => {
+    event.preventDefault()
+    props.editReview(editedReview, props.id)
+    props.toggleEdit()
```

Comment:
`editReview` should return `true` or `false` and based on that we should invoke `toggleEdit`

```
if (editReview()) {
  toggleEdit()
}
```